File: interactive/php/extract-method-duplicate_after.php
class Order {
  function validate() {
    // validate customer
    $this->getInfo($customer);
  }

  function dataProcess() {
    $this->getInfo($customer);
    // data process
  }

  function getInfo($customer) {
    echo $this->$cusomer->getName();
    echo $this->$cusomer->getEmail();
  }
}

File: interactive/csharp/extract-method-duplicate_after.cs
class Order 
{
  void validate() 
  {
    // validate customer
    getInfo(customer);
  }

  void dataProcess() 
  {
    getInfo(customer);
    // data process
  }

  void getInfo(Customer customer) 
  {
    Console.WriteLine(cusomer.getName);
    Console.WriteLine(cusomer.getEmail);
  }
}
